src/components/header/header.scss

```diff
@@ -1,19 +1,20 @@
 @use '../../app/globals';
 
 body {
+  header {
+    // box-shadow property will have an offset on either side if the header
+    // element is not 105% width
+    width: 105%;
```

Comment:
It causes a horizontal scroll on the page
I think it would be better to just adjust the box-shadow

Reply:
Good catch on the horizontal scroll